composer.json

```diff
@@ -64,7 +64,8 @@
         "symfony/symfony": "2.8.10",
         "symfony/phpunit-bridge": "2.8.10",
         "symfony/monolog-bundle": "2.8.10",
-        "jackalope/jackalope": "1.2.8 || 1.3.0 || 1.3.1"
+        "jackalope/jackalope": "1.2.8 || 1.3.0 || 1.3.1",
+        "phpcr/phpcr-utils": "1.2.1 - 1.2.10 || 1.3.1 || 1.3.2"
```

Comment:
Why do we have such a strange version constraint here? Do you think 1.3.3 will already break again? Same for `jackalope/jackalope`, although that was already like that before.

Reply:
Ah, sry, didn't see that this is the conflict section :see_no_evil: